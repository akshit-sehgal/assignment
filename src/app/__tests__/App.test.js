import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('Testing <App />', () => {

    it('should render the default view', () => {
        const wrapper = shallow(<App />);
        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('BrowserRouter').length).toEqual(1);
        const routesWrapper = wrapper.find('Routes');
        expect(routesWrapper).toMatchSnapshot();
        expect(routesWrapper.length).toEqual(1);
        expect(routesWrapper.dive().find('Route').length).toEqual(2);        
    });

});