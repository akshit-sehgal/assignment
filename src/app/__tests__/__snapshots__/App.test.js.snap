// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <App /> should render the default view 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <BrowserRouter>
    <Routes />
  </BrowserRouter>
</Provider>
`;

exports[`Testing <App /> should render the default view 2`] = `<Routes />`;
